### ISSUE TITLE
Wienersachtext formula reporting wrong results

### ISSUE THREAD

**Issue by alxwrd** [shivam5992/textstat#156]
The Wienersachtext formula is failing a test. The reported figure is wrong, and it appears to be missing rounding.

```plain
_____________________________________________ test_wienersachtext_formula ______________________________________________

    def test_wienersachtext_formula():
        sample_text = 'Alle meine Entchen schwimmen auf dem See, Köpfchen unters Wasser, Schwänzchen in die Höh.'
        wstf = textstat.wiener_sachtextformel(sample_text, variant=1)

>       assert wstf == 3.8
E       assert 6.300800000000001 == 3.8

test.py:482: AssertionError
```

**Comment by Clickedbigfoot** [comment]
#163 I added rounding to the function and added an extra test, but also had it call set_lang() to set Pyphen to German. It doesn't seem to calculate the correct syllables otherwise. 
The extra test I added is a sentence I pulled from a German newsite

**Comment by alxwrd** [comment]
Closed with #163
